deploy/roles/matlab/defaults/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+matlab_install_directory:   /usr/local/MATLAB
+# matlab_install_files:       /usr/local/MATLAB_install
+matlab_version:             R2018b
+matlab_iso1_mount:          iso1
+matlab_iso2_mount:          iso2
+matlab_license_folder:      licenses
+matlab_combined_mount:      installer
+matlab_output_location:     output
+install_matlab:           False
+# matlab_tarball:                      matlab_R2018a_glnxa64.zip
+
+# matlab_local_bin_directory: /home/dtwork/tools
+# matlab_local_bin_directory: /artifacts
+# matlab_zip_file:            matlab_R2018b_glnxa64.zip
+matlab_iso1:                R2018b_glnxa64_dvd1.iso
+matlab_iso2:                R2018b_glnxa64_dvd2.iso
+
+
+matlab_installation_key:     installkey
+matlab_outputfile:           /usr/local/MATLAB/R2018b/output/mathworks.log
+matlab_license_location:     /usr/local/MATLAB/R2018b/licenses/license.lic
+matlab_network_license_file: /usr/local/MATLAB/R2018b/licenses/network.lic
+matlab_license_file:         proprietary_license_info
+matlab_license_server:       server
+matlab_license_daemon:       daemon
+matlab_license_symlink_src:  etc/lmboot
```

Comment:
@stirlingv In addition to this symlink, there are two others: /etc/lmboot, and /etc/lmdown, that should be linked to $MATLAB_PATH/etc/lmboot and $MATLAB_PATH/etc/lmdown respectively.  These are used by the "flexnet" init.d script located at /etc/init.d/flexnet, which was created by Matlab installer and starts at system boot.  Since these links were not updated, the 2015 version of the license manager was running and folks couldn't start Matlab (due to patching).